README.md

```diff
@@ -24,86 +26,218 @@ After you're finished please remove all the comments and instructions!
 
-## 🛠 Built With 
-<a name="built-with">html</a>
-<a name="built-with">css</a>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">n/a</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **It show case my project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> https://massazoum.github.io/Portfolio-setup-and-mobile-first/
+- [Live Demo Link](https://massazoum.github.io/Portfolio-setup-and-mobile-first)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
 
-Run git clone https://github.com/massazoum/Portfolio-setup-and-mobile-first.git
+### Prerequisites
+
+In order to run this project you need:
+- vs code 
+- github
+- laptop
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
 
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
-## 👥 Authors <a name="authors">Massazoumanigui</a>
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 **Massa zoumanigui**
 
 - GitHub: [@massazoum](https://github.com/massazoum)
-- Twitter: [@zoumaniguimass1](https://twitter.com/zoumaniguimass1)
+- Twitter: [@@zoumaniguimass1](https://twitter.com/zoumaniguimass1)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/massa-zoumanigui-1aba4525a)
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-## 🤝 Contributing
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Animation**
+- [ ] **Social media icons**
+- [ ] **Responsive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
 ## ⭐️ Show your support <a name="support"></a>
 
-> If you like this project reach out me ...
-> 
+> Write a message to encourage readers to support your project
```

Comment:
Kindly take away these template guideline statements as they make your readme look more like a template, giving it a less professional look😉